structure/04 Triathlon/01Trainer/2019_10_02TriathlonTrainer.html

```diff
@@ -0,0 +1,8 @@
+<html>
```

Comment:
```suggestion
<html>
<body>
```

Please create more valid html and include body tags 😉